.gitignore

```
@@ -1,6 +1,7 @@
 ### Project ###
 
 packages/
+**/.rocks
```

It works on any subdirectory
```suggestion
.rocks
```